src/parsers/manifestjson.js

```diff
@@ -129,6 +129,17 @@ export default class ManifestJSONParser extends JSONParser {
       this.validateIcons();
     }
 
+    if (this.parsedJSON.background) {
+      if (this.parsedJSON.background.scripts) {
+        for (let i = 0; i < this.parsedJSON.background.scripts.length; i++) {
+          this.validateExists(this.parsedJSON.background.scripts[i], 'script');
+        }
```

Comment:
We recently started using more `forEach` etc so this could be something like

```javascript
this.parsedJSON.background.scripts.forEach(() => {
    this.validateExists(...)
});
```

Reply:
wfm